tests/integration/update_cluster/many-addons-ccm-irsa/data/aws_s3_bucket_object_minimal.example.com-addons-aws-cloud-controller.addons.k8s.io-k8s-1.18_content

```diff
@@ -20,12 +20,13 @@ spec:
     spec:
       containers:
       - args:
-        - --v=2
+        - --allocate-node-cidrs=true
         - --cloud-provider=aws
-        - --cluster-name=minimal.example.com
         - --cluster-cidr=172.20.0.0/16
-        - --allocate-node-cidrs=true
+        - --cluster-name=minimal.example.com
         - --configure-cloud-routes=false
+        - --leader-elect=true
```

Comment:
Kind of scary to think of previous HA clusters not having leader election on ... oh well.

Reply:
Otherwise, only the ordering of the flags was changed.